config/wdio/visualRegressionConf.js

```diff
@@ -27,8 +27,9 @@ function createTestName(fullName) {
 }
 
 function getScreenshotName(context) {
+  const { name } = (context.options || {});
   const parentName = createTestName(context.test.parent);
-  const testName = createTestName(context.test.title);
+  const testName = createTestName(name || context.test.title);
```

Comment:
If `context.objects` does not exist, and we attempt to pull `name`, we can get the error: 
> cannot get name from undefined.

By falling back to `{}` we will try to pull `name` from an empty object which will result in `name` being undefined.

If `name` is undefined, then `context.test.title` will be used.

The reason we need this change are for the new `validates` test options. Currently Terra.should.matchScreenshot is actually the following:
```js
it(`[${testName}] be within the mismatch tolerance`, () => {
   runScreenshotComparision()
})
```
For this example test spec called `check-spec.js`:
```js
describe('check test', () => {
   before(() => browser.url('test.com'));
   Terra.should.matchScreenshot();
});
```
results in name like:
`reference/en/chrome_huge/check-spec/check_test[default].png`

With adding Terra.validates.screenshot()  we would like to keep naming consistent. This would look like:
```js
describe('check test', () => {
   it('goes to page & check screenshot', () => {
      browser.url('test.com');
     Terra.validates.screenshot();
   });
});
```
Under the hood I pass `name` option for both the it and validates option to ensure the naming for that test is consistent.

I would suspect most of the time `context.test.title` will never be used, however if people did want to skip the terra helpers and use the visual regression service directly, we will want to fallback to the test name, otherwise we will result in `undefined`.